coen320_main

```diff
@@ -0,0 +1,389 @@
+// TODO: Implement a second array, and test with it
```

Comment:
Why are you repushing this?

Reply:
When i updated my branch i guess it did soemthing

Reply:
You added all files into a commit, thats why

Reply:
oh well, still the same file tho so...